tests/conversion_test.py

```diff
@@ -191,21 +190,6 @@ def test_pytket_qir_10() -> None:
     check_qir_result(result, "test_pytket_qir_10")
 
 
-def test_pytket_qir_11() -> None:
-    # test copybits op
-    c = Circuit(1, name="test_classical")
-    a = c.add_c_register("a", 2)
-    b = c.add_c_register("b", 4)
-
-    c.add_c_copyreg(a, b)
-
-    result = pytket_to_qir(c, name="test_pytket_qir_10", qir_format=QIRFormat.STRING)
-
-    check_qir_result(
-        result, "test_pytket_qir_10"
-    )  # should be identical to the testcase above
-
-
```

Comment:
Note I deleted this test because with `init_reg`, the output is different from `test_pytket_qir_10`